shared/common-adapters/popup-menu.desktop.js

```diff
@@ -51,6 +53,10 @@ class Menu extends Component<void, Props, void> {
   }
 }
 
+PopupMenu.defaultProps = {
+  visible: true,
```

Comment:
Some of the component's consumers hide the component behind a `SOME_VISIBILITY_BOOLEAN && <PopupMenu ... />` (profile) while others use the `visible` prop (folders, devices, menubar). I didn't want to change the `visible` prop consumers but I wanted to use the first method in my use cases, and adding the `visible: true` as default made that easier. I can remove the `visible` support entirely so that the component matches the standard style of our other stuff.